src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -94,15 +100,49 @@ export class SkyDatepickerInputDirective implements
     this._disabled = value;
   }
 
+  private get modelValue(): Date {
```

Comment:
Should we always be declaring setters first?
https://host.nxt.blackbaud.com/skyux-team-docs/style

Reply:
Good question. In non-Angular code, the convention seems to be that `get` comes first, but we made the distinction for `Input` that `set` comes first (I was under the assumption that _unless_ it was an `Input`, `get` should come first). I can see that being a confusing style distinction, however. What do you think?

Reply:
Yeah... that's a bit confusing. Let's chat about this as a team and we can decide - it doesn't need to hold up this PR. Resolving this thread.